sections/faqs/extend-and-styled-difference.js

```diff
@@ -0,0 +1,25 @@
+import md from 'components/md'
+
+const CSSFrameworks = () => md`
+  ## What is the difference between \`Comp.extend\` and \`styled(Comp)\`?
+
+
+  You have two choices to add additional styles to a component or a styled component,
+  one is using [\`Comp.extend\`](/docs/basics#extending-styles) and the other is wrapping your
+  component in the styled method like this: [\`styled(Comp)\`](/docs/basics#styling-any-components).
+
+  The styled method works perfectly on all of your own or any third-party components
+  as well, but the extend method can only be used on a styled component only.
+
+  Styled method always generates a real stylesheet with classes. The classnames are
+  then passed to the React component (including third party components) via the className prop.
+  Calling \`extend\` creates a new stylesheet by extending the old one, and thus doesn't
```

Comment:
~stylesheet~ component styles / styling / rules

~extending~ combining the old set of rules and the ones that it receives

also duplicates previous explanations btw. It'd be detrimental and confusing to have too many duplicating explanations I'm afraid :/

Reply:
It's not duplicate as we've replaced that from that section
with this:
> You should only use \`Comp.extend\` if you know that \`Comp\` is a styled component.
If you're importing from another file or a third party library, prefer to use
\`styled(Comp)\` as it accomplishes the same thing but works with *any* React
 component. Read more about [What is the difference between \`Comp.extend\` and \`styled(Comp)\`?](/docs/#what-is-the-difference-betweencompextendandstyled(comp)?)